src/MakeModel.py

```diff
@@ -103,7 +103,7 @@ class ModelerException(Exception):
 class Modeler:
     def __init__(self, debug=False,
                  TelluricModelingDirRoot=DEFAULT_TELLURICMODELING,
-                 nmolecules=12):
+                 nmolecules=12, printoutshow=True):
```

Comment:
Could you rename this to something more informative, like `print_lblrtm_output`?